spec/support/initializers/capybara.rb

```diff
@@ -21,7 +21,16 @@
   end
 
   config.before(:each, type: :system, js: true) do
-    driven_by :headless_chrome
+    if ENV["SELENIUM_URL"].present?
+      # Support use of selenium-docker
```

Comment:
Technically this would work for other scenarios as well, like running it on another machine. So if we introduce this, maybe we can add another option and also change the next line to this:

```ruby
Capybara.server_host = ENV.fetch("CAPYBARA_SERVER_HOST") { "0.0.0.0" }
```

Reply:
Updated!